fellow-submissions/peter-rose.md

```diff
@@ -0,0 +1,148 @@
+# Problem Set 2.4 - Scope
+
+**1. How do variables declared with `let`, `const`, and `var` differ? Be sure to touch on _scope_, _reassignment_, and _hoisting_ for each.**
+
+<!---->
+   Firstly, const cannot change its value. Thats the purpose of it so that variable will never be able to reassign to a new value. Unlike
```

Comment:
A small adjustment
> const cannot change its value
... instead say, "variables declared with `const` cannot be **reassigned**.

Also, be sure to use backticks around all variable names and values so that they are formatted as code snippets.